internal/engine/extractors.go

```diff
@@ -45,6 +45,18 @@ func extractImageTargets(specs []model.TargetSpec) []model.ImageTarget {
 	return iTargets
 }
 
+func extractDockerComposeTargets(specs []model.TargetSpec) []model.DockerComposeTarget {
+	iTargets := make([]model.DockerComposeTarget, 0)
```

Comment:
`iTarget` is shorthand for "image targets", call this something else

Reply:
done